### PR TITLE
Rephrase logical clauses in docstrings

### PR DESCRIPTION
**Pull request by qagren**
**Problem**

Two docstrings incorrectly phrased logical conditions. 

**Solution**

Suggest new formulations that match the logical branching conditions.

**Acceptance Criteria**

Commit messages describe the rational of the changes in detail.

### CODE REVIEW

**Comment by qagren** [comment]
Should I need to worry about the CI failing ?

File: import_export/resources.py [hunk 1]
```diff
@@ -489,7 +489,7 @@ def after_delete_instance(self, instance, dry_run):
     def import_field(self, field, obj, data, is_m2m=False):
         """
         Calls :meth:`import_export.fields.Field.save` if ``Field.attribute``
-        and ``Field.column_name`` are found in ``data``.
+        is specified, and ``Field.column_name`` is found in ``data``.
```
Comment (manelclos) [inline import_export/resources.py:492]: agree, current despcription is not correct.

File: import_export/resources.py [hunk 2]
```diff
@@ -546,8 +546,8 @@ def skip_row(self, instance, original):
         """
         Returns ``True`` if ``row`` importing should be skipped.
 
-        Default implementation returns ``False`` unless skip_unchanged == True,
-        or skip_diff == True.
+        Default implementation returns ``False`` unless skip_unchanged == True
+        and skip_diff == False.
```
Comment (manelclos) [inline import_export/resources.py:550]: I see what you mean here. The thing is that the default value for `skip_unchanged` is `False`, and the default value for `skip_diff` is `False`. So changing any of those variable values will skip the default implementation and check for changes to decide if the row has to be skipped. I'd say you are right here too, but we should emphasise default values, something like this:

    Default implementation returns ``False`` unless skip_unchanged == True (defaults to False)
    and skip_diff == False (default).

Comment (qagren) [reply import_export/resources.py:550]: Thanks for your feedback. I agree, howevery I think it is quite a lot of information for one sentence. How about:
```
Default implementation returns ``False`` unless skip_unchanged == True 
and skip_diff == False. 

Left unspecified, skip_diff``  and skip_unchanged`` both default to ``False``, 
and rows are never skipped. 
```

Comment (manelclos) [reply import_export/resources.py:550]: Much better!

Comment (qagren) [reply import_export/resources.py:550]: Changes are pushed!

**Review by manelclos — APPROVED** [review]

**Comment by manelclos** [comment]
@qagren CI config and tests have been fixed since you made the PR, if you rebase to current `master` branch tests should pass

**Review by andrewgy8 — APPROVED** [review]
Thank you @qagren !

**Comment by qagren** [comment]
You're very welcome, thanks to *you* for the great work!

**Pull request merged**